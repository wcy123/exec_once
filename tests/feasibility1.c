#include <stdio.h>
__attribute__((constructor))
static void foo() 
{
    printf("Hello World from a constructor\n");
}

int main(int argc, char *argv[])
{
    printf("Hello World");
    return 0;
}
