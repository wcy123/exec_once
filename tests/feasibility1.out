Hello World from a constructor
Hello World